### PR TITLE
data_model: implement copyright date transformation

### PR DESCRIPTION
**Pull request by reropag**
* updates 300 and 10k fixtures files
* implements copyright date transformation

Co-Authored-by: Gianni Pante <gianni.pante@rero.ch>
Co-Authored-by: Peter Weber <peter.weber@rero.ch>

### CODE REVIEW

**Comment by BadrAly** [comment]
If possible update the unittest fixtures and tests @ test_documents_jsonschema.py

File: rero_ils/dojson/cli.py [hunk 1]
```diff
@@ -18,6 +18,8 @@
 """Click command-line utilities."""
 
 
+import json
```
Comment (jma) [inline rero_ils/dojson/cli.py]: why?

File: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json [hunk 2]
```diff
@@ -167,6 +167,12 @@
         }
       }
     },
+    "copyrightDate": {
+      "title": "Copyright Date",
+      "description": "Copyright Date of the resource.",
+      "type": "string",
```
Comment (jma) [inline rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json]: is it a date or a string? Can we put more constraints on this fields regex or other?

File: rero_ils/dojson/cli.py (hunk 1 above)
Comment (reropag) [reply rero_ils/dojson/cli.py]: without this import, the generation of documents will not work

File: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json (hunk 2 above)
Comment (reropag) [reply rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json]: regex pattern added

**Comment by reropag** [comment]
> If possible update the unittest fixtures and tests @ test_documents_jsonschema.py

Done

**Pull request merged**